### ISSUE TITLE
I think we should include the angular un-minified and maps with the distribution...

### ISSUE THREAD

**Issue by iJungleboy** [2sic/2SexyContent#142]
Because IMHO many would benefit from seeing the internals of angularjs, and including these has no security impact but would make live-development much easier. 

Pro
* easier debugging
* better discovery on how angularjs works
* less errors when debugging (can't find .min...)
* ensure that the files are updated when we replace the angularjs version (so it guarantees that the .map file is the same version as the angular distribution, and not that the user added a .map and we updated the angularjs
* could easily be removed one day if we decide it was a bad idea, as all "normal" users wouldn't be affected

Cons
* more files in the distribution
* larger distribution size

**Comment by iJungleboy** [comment]
New - better idea: let's change the min-file to point to the map on a CDN. I tried it w/google and it works. In my test I replaced the last line which was originally
//# sourceMappingURL=angular.min.js.map

with
//# sourceMappingURL=//ajax.googleapis.com/ajax/libs/angularjs/1.3.9/angular.min.js.map

works like a charm :)!

Of course we will always have to update the version, but I believe this is the way to go!
